src/Scroll-Bar.scss

```diff
@@ -0,0 +1,34 @@
+@import 'style/constants';
+
+.MissionScroll{
+
+    height:200px;
+
+    .card{
+        display: flex;
+        background-color: #2c3e50;
```

Comment:
We should probably move these colours into our constants file, so that we can reuse them across the site.

Same for the red below